ReText/editor.py

```diff
@@ -370,14 +370,28 @@ def resizeEvent(self, event):
 	def highlightCurrentLine(self):
 		if globalSettings.relativeLineNumbers:
 			self.lineNumberArea.update()
-		if not globalSettings.highlightCurrentLine:
+		if globalSettings.highlightCurrentLine == 'disabled':
 			return self.setExtraSelections([])
-		selection = QTextEdit.ExtraSelection();
-		selection.format.setBackground(colorValues['currentLineHighlight'])
-		selection.format.setProperty(QTextFormat.FullWidthSelection, True)
-		selection.cursor = self.textCursor()
-		selection.cursor.clearSelection()
-		self.setExtraSelections([selection])
+		elif globalSettings.highlightCurrentLine == 'cursor-line':
+			selection = QTextEdit.ExtraSelection();
+			selection.format.setBackground(colorValues['currentLineHighlight'])
+			selection.format.setProperty(QTextFormat.FullWidthSelection, True)
+			selection.cursor = self.textCursor()
+			selection.cursor.clearSelection()
+			self.setExtraSelections([selection])
+		elif globalSettings.highlightCurrentLine == 'wrapped-line':
+			selections = [QTextEdit.ExtraSelection(),QTextEdit.ExtraSelection()]
```

Comment:
code style: add a space after comma

```suggestion
			selections = [QTextEdit.ExtraSelection(), QTextEdit.ExtraSelection()]
```